pkg/apis/sriovnetwork/v1/sriovnetwork_types.go

```diff
@@ -16,6 +16,8 @@ type SriovNetworkSpec struct {
 	// +kubebuilder:validation:Minimum=0
 	// +kubebuilder:validation:Maximum=4096
 	Vlan             int    `json:"vlan,omitempty"`
+	Spoof            string `json:"spoof,omitempty"`
+	Trust            string `json:"trust,omitempty"`
 }
 
```

Comment:
maybe we should add state { auto | enable | disable} ?
- auto: a reflection of the PF link state (default)
- enable: lets the VF to communicate with other VFs on this host even if the PF link state is down
- disable: causes the HW to drop any packets sent by the VF.

Reply:
This is a good one. I'm going to add parameters for what will be supported by SRIOV-CNI in this PR: https://github.com/intel/sriov-cni/pull/92

Reply:
I think it may be better to use value instead of pointer here. So that develop can use this struct without considering deepcopy problem.

Reply:
This was originally a value, I changed it due to that I'd like to differentiate between unset and zero values of boolean type, only insert the config in CNI once it's not nil.